### DISCUSSION TITLE
Remove share button

### DISCUSSION THREAD

**Discussion by aluiss** [requarks/wiki#3826] [General]
Hi, good morning!

Can i remove share and print buttons from page views?

Thanks!

**Comment by aluiss** [comment]
> Hi, good morning!
> 
> Can i remove share and print buttons from page views?
> 
> Thanks!

I has use a CSS inject. Thanks!

**Reply by hicks0r** [reply]
Hi!. Could you show me the code the hide it via CSS? Thank you.

**Reply by P4mes4n** [reply]
This works for us (on a german install, you probably have to inspect element to see the aria-label on your installation):
```
button[aria-label=Teilen] {
display: none
}
```

**Comment by lunardigs** [comment]
Hello,
This is something we'd like too.
